state/status.go

```diff
@@ -295,185 +294,6 @@ func PruneStatusHistory(mb modelBackend, maxHistoryTime time.Duration, maxHistor
 	// NOTE(axw) we require a raw collection to obtain the size of the
```

Comment:
Should this comment block move to pruneCollection() and the use of GetRawCollection()?